jQuery/styles.css

```diff
@@ -1,27 +1,19 @@
-body {
-  margin: 0;
-  padding: 0;
-}
-
 h1 {
-  margin: 20px 5%;
+  font-size: 2em;
+  margin-bottom: 20px;
 }
 
 #hh-menu {
   border-right: solid 1px black;
   position: fixed;
   width: 240px;
   height: 100%;
-}
 
-#hh-menu > select {
-  width: 80%;
-  margin-left: 15%;
-  margin-right: 5%;
+  padding: 20px;
 }
 
-#hh-menu > label {
-  margin-left: 5%;
+.menu-select {
+  width: 90%;
 }
 
 #hh-content {
```

Comment:
I think we might not be on the same page here. I'm not trying to center anything at the moment, my menu is ```position: fixed``` to the left side of the window at a static width of 240px, so the ```margin-left: 240px``` here is just to ensure nothing in ```#content``` overlaps with the menu. The intent of the padding here wasn't to center the content, just to give the ```Hello wolr``` some breathing room and will almost certainly change once we start working on the actual content of the page